cmds/core/head/head.go

```diff
@@ -26,7 +26,7 @@ func run(stdout io.Writer, stderr io.Writer, bytes, count int, files ...string)
 	var buffer []byte
 	if bytes > 0 {
 		printBytes = true
-		buffer = make([]byte, bytes)
```

Comment:
please accept my apologies. My brain read "ReadFull" as "ReadFile" -- !@#$#$!

So maybe the bytes parameter is ok. Sorry :-(

Reply:
No worries at all. I understood your comment in following way. If user pass as bytes parameter more memory that is currently have this program will fail. And fancied such use-case  `head -c=10000000 file > f` and with 4K buffer it should work. What do you think?